### PR TITLE
Scg news task 8

### PR DESCRIPTION
**Pull request by SpencerCGriffiths**

### CODE REVIEW

File: app/__tests__/app.test.js [hunk 1]
```diff
@@ -187,3 +187,72 @@ describe("GET /api/articles/:article_id/comments", () => {
...
+        test("200: votes can be reduced from one of the articles", () => {
+            const updatedVotes = { 
+                inc_votes: -25
+            }
+            return request(app)
+            .patch("/api/articles/1")
+            .send(updatedVotes)
+            .expect(200)
+            .then(({body}) => {
+                console.log(body.updatedArticle)
```
Comment (MKD01) [inline app/__tests__/app.test.js]: Dont forget to remove console.logs

File: app/__tests__/app.test.js [hunk 2]
```diff
@@ -187,3 +187,72 @@ describe("GET /api/articles/:article_id/comments", () => {
...
+                    author: 'butter_bridge',
+                    body: 'I find this existence challenging',
+                    created_at: '2020-07-09T20:11:00.000Z',
+                    votes: 75,
+                    article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+                  })
+            })
+        })
+
+    })
```
Comment (MKD01) [inline app/__tests__/app.test.js:452]: Really good testing so far, but there are some error handling tests that should be added for this endpoint as well.

What should happen when a user makes a request to an article that doesnt exist?
What should happen when a user makes an invalid request? i.e. /api/articles/banana
What should happen when a user makes a valid request but inc_votes is missing/invalid in the provided body?

**Review by MKD01 — CHANGES_REQUESTED** [review]
Really good work so far, but I would like to see those error handling tests added before you merge this in, so please re-submit this pr once you have added those in

**Review by aloush-dev — APPROVED** [review]
Really good job implementing feedback.
Nice testing!

Merge when ready.

**Pull request merged**